pkg/apis/policy/types.go

```diff
@@ -157,7 +157,7 @@ type PodSecurityPolicySpec struct {
 	// To allow all capabilities you may use '*'.
 	// +optional
 	AllowedCapabilities []api.Capability
-	// Volumes is a white list of allowed volume plugins. Empty indicates that
+	// Volumes is an allowed list of allowed volume plugins. Empty indicates that
```

Comment:
Either "an allowlist of" or "a list of allowed" but not "an allowed list of allowed", that's redundant :)

Reply:
@lavalamp Thanks ! I replace all "an allowed list of allowed" to "an allowlist of".